ui/tests/acceptance/transit-test.js

```diff
@@ -177,24 +186,25 @@ const testConvergentEncryption = async function(assert, keyName) {
       context: 'secret 2',
       encodePlaintext: true,
       encodeContext: true,
-      decodeAfterDecrypt: true,
+      decodeAfterDecrypt: false,
```

Comment:
We updated all of these to not decode after decrypt since the base64 toggle was removed -- we should probably remove that piece of the test now since none of these use it